src/routes/guilds/guild/leaderboard.ts

```diff
@@ -0,0 +1,83 @@
+import { Route, RouteStore } from 'klasa-dashboard-hooks';
+import ApiRequest from '../../../lib/structures/api/ApiRequest';
+import ApiResponse from '../../../lib/structures/api/ApiResponse';
+import { Snowflake } from 'discord.js';
+import { LeaderboardUser } from '../../../lib/util/Leaderboard';
+import { isNumber } from '@klasa/utils';
+
+export default class extends Route {
+
+	public constructor(store: RouteStore, file: string[], directory: string) {
+		super(store, file, directory, { route: 'guilds/:guild/leaderboard' });
+	}
+
+	public async get(request: ApiRequest, response: ApiResponse) {
```

Comment:
Shouldn't this be RateLimited?

Reply:
Uhh, care to elaborate?

Reply:
All the routes have a @ratelimit decorator attached to the methods which rateLimit the requests to prevent spam, this should follow that pattern